.github/CODEOWNERS

```diff
@@ -81,7 +81,7 @@ android/src/fdroid/play/listings/mr-IN/ @organicmaps/translations-mr
 android/src/fdroid/play/listings/pt-BR/ @organicmaps/translations-pt
 android/src/fdroid/play/listings/ru-RU/ @organicmaps/translations-ru
 android/src/fdroid/play/listings/tr-TR/ @organicmaps/translations-tr
-android/src/fdroid/play/listings/zh-CN/ @organicmaps/translations-zh
+android/src/fdroid/play/listings/zh-CN/ @organicmaps/translations-zh-Hans
```

Comment:
Sholdn't it be listings/zh/ for simplified Chinese?

Reply:
What should be other values for traditional one? HK, MO, TW? Should we also hard-code them?

Reply:
> Sholdn't it be listings/zh/ for simplified Chinese?

Please don't start this flame war. Let's follow existing conventions in Android/iOS. Simplified (Mainland) will be zh-Hans, Traditional (HK, Macao, Taiwan) - zh-Hant. No discrimination. No decision we need to make on our own.

> What should be other values for traditional one? HK, MO, TW? Should we also hard-code them?

I am not a big expert in Chinese, but as far as I remember, Hong Kong, Macao, and Taiwan use **traditional** Chinese (zh-Hant), whereas Mainland, Singapore and few other places use Simplified. There are can be some local differences, like between British and American English variations. The difference between Simplified and Traditional is **huge**.

@organicmaps/translations-zh-Hans, @organicmaps/translations-zh-hant, please correct me if I am wrong.